packages/core/src/test/credentials/sso/ssoAccessTokenProvider.test.ts

```diff
@@ -233,6 +235,65 @@ describe('SsoAccessTokenProvider', function () {
             })
         })
 
+        describe('telemetry: ReAuthReason during SSO flow', () => {
+            // Note in the future we should use an `identifier` instead of `startUrl` for this test
+            beforeEach(() => {
+                setupFlow()
+                stubOpen()
+            })
+
+            it('does not clear the reAuthReason on failed login', async () => {
```

Comment:
we have existing tests for the various auth cases. can we just add asserts about reAuth in those existing tests? reauth itself is not a topic, it's just a field. 

It helps connect concepts if auth fields are all tested in the auth tests, instead of having separate test groups for every piece of auth state.

Reply:
Updated